docs/creating-editing-and-removing-specialist-document-types-and-finders.md

```diff
@@ -70,20 +70,29 @@ The email sign up page is rendered by [Finder Frontend](https://github.com/alpha
 
 If your email sign up page should have checkboxes (e.g. [cma-cases](https://www.gov.uk/cma-cases/email-signup)), you will need to edit email-alert-api by adding the new tags to [valid_tags.rb](https://github.com/alphagov/email-alert-api/blob/3e0018510ea85f5d561e2865ad149832b94688a1/lib/valid_tags.rb#L2).
 
-## 5. Deploy and publish <a id="deploy-and-publish"></a>
-
-To deploy:
+## 5. Deploy a finder to the draft stack (for previewing)
 
+To deploy a new finder for previewing:
+   1. Ensure the finder target_stack is set to `draft`
    1. Deploy Publishing API (if you haven't already), Specialist Publisher and Search API. 
       - Ensure you deploy Publishing API first, to avoid schema validation errors.
       - Also deploy Email Alert API if you have made changes to it.
-   2. [Reindex the govuk Elasticsearch index](https://docs.publishing.service.gov.uk/manual/reindex-elasticsearch.html#how-to-reindex-an-elasticsearch-index).
+   1. [Reindex the govuk Elasticsearch index](https://docs.publishing.service.gov.uk/manual/reindex-elasticsearch.html#how-to-reindex-an-elasticsearch-index).
```

Comment:
Hmmm - given it's only the finder 'skeleton' that's previewable, I'm not sure there's much point to the indexing here. Should this step (and its substeps) be moved into the publish section instead?